shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -69,6 +101,14 @@ protected int getNumOfArgs() {
 
   @Override
   public int run(CommandLine cl) throws IOException {
+    String[] args = cl.getArgs();
+    if (cl.hasOption("h")
```

Comment:
validate that the options used match the subcommand

Reply:
Make sense!
Changed to 
``` java
// Only capacity category has [category args]
    if (!command.equals(Command.CAPACITY)) {
      if (cl.getOptions().length > 0) {
        throw new InvalidArgumentException(
            String.format("report %s do not support arguments: %s",
                command.toString().toLowerCase(), cl.getOptions()[0].getOpt()));
      }
    }
```